frontend/src/components/LandingPage/Tools/Tools.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import "./Tools.css"
+import {
+    Card, CardImg, CardBody,
+    CardTitle, CardSubtitle
+  } from 'reactstrap';
+
+const react_logo = "/react-logo.gif"
+
+class Tools extends React.Component {
```

Comment:
Since we are displaying something to the user only, what do you think about changing this to a functional component instead?